api/graphql/reservation_units/reservation_unit_serializers.py

```diff
@@ -365,6 +379,15 @@ def validate(self, data):
                     "nameFi is required for draft reservation units."
                 )
 
+        if data.get("payment_types"):
```

Comment:
If this is still relevant I would make it as own method.

Reply:
It is not relevant from validation point of view so I removed it. However, I just realized how awful to provided error is:
```
{
  "data": {
    "createReservationUnit": {
      "errors": [
        {
          "messages": [
            "0"
          ]
        }
      ],
      "pk": null
    }
  }
}
```

It basically says the index of array where the invalid value is and nothing more. I think this is not good. I could restore the custom validation and make it as own method.

Reply:
Actually, the build-in validation kicks in before `validate()` method is called 😢
Any wild ideas?

Reply:
Wild idea would be make own field inheriting the `ListField` and do some overwriting there :thinking:

Reply:
It works! 🥳